CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 # Changelog
 All notable changes to this library are documented in this file. 
 
-# [3.1.0] - UNREALEASED
+# [4.0.0]
```

Comment:
Bumping the major version number requires package name change to `github.com/tuneinsight/lattigo/v4` (along with associated changes).